components/landing/QuickStart/StartLocally.vue

```diff
@@ -15,29 +15,30 @@
     </p>
     <div class="start-locally__options">
       <div
-        v-for="optionsBlock in installOptions"
-        :key="optionsBlock.alias"
+        v-for="choicesGroup in installChoices"
+        :key="choicesGroup.id"
       >
         <h4 class="start-locally__option-title">
-          {{ optionsBlock.title }}
+          {{ choicesGroup.title }}
         </h4>
         <cv-button-set :class="`start-locally__options-group`">
           <cv-button
-            v-for="option in optionsBlock.elements"
-            :key="option.label"
-            :title="option.label"
+            v-for="option in choicesGroup.options"
+            :key="option"
+            :title="option"
             class="start-locally__option"
-            :class="{ 'start-locally__option_active': option.active }"
+            :class="{ 'start-locally__option_active': isActive (choicesGroup.id, option) }"
+            @click="selectOption(choicesGroup.id, option)"
```

Comment:
Non-blocking: I'd name the function `selectChoiceOption` giving a hint that you need to pass a choice and an option and pass the whole choice so the `id` is inspected inside the function. That way you need to know nothing about the choice group, determining its id is completely abstract.
```suggestion
            @click="selectChoiceOption(choicesGroup, option)"
```

Reply:
Agree to inspect the `id` inside the function. 

I think `selectOption` represents better than  `selectChoiceOption` what the function does, so I will keep it :)